tools/src/main/java/org/apache/kafka/tools/CommandLineUtils.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.tools;
+
+import net.sourceforge.argparse4j.inf.ArgumentParser;
+import org.apache.kafka.common.utils.Exit;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Helper functions for dealing with command line utilities
+ */
+public class CommandLineUtils {
+
+    /**
+     * Print usage and exit
+     *
+     * @param parser    parser used for command line arguments
+     * @param message   message to print as an error before die
+     * @throws IOException
+     */
+    public static void printUsageAndDie(ArgumentParser parser, String message) throws IOException {
+        System.err.println(message);
+        parser.printHelp();
+        Exit.exit(1);
```

Comment:
Checked out and ran your code, started to write unit tests around it. Generally my opinion is that instead of dying we should create an exception type and throw it here. At the top level in our main method we might catch it, print its error message and the help.

This way this class would be more reusable. As per KIP-4, the end goal is not just the refactor of these commands but eventually creating a "kafka shell" which allows the nice running of these commands. Although this shell in itself would require a KIP for the details, I think it's better if we keep it in mind when designing the new commands :).